src/scripts/main.js

```diff
@@ -1,3 +1,324 @@
+const employeesTableList = employeesTable.querySelector('tbody');
+const tableContainer = document.createElement('div');
+
+tableContainer.style.display = 'flex';
+
+page.append(tableContainer);
+tableContainer.append(employeesTable);
+
+/* ============== EMPLOYEE FORM CREATION ============== */
+
+tableContainer.insertAdjacentHTML('beforeend', `
+  <form class="new-employee-form">
+  <label>Name:
+    <input name="name" type="text" data-qa="name">
+  </label>
+  <label>Position:
+    <input name="position" type="text" data-qa="position">
+  </label>
+  <label>Office:
+    <select name="office" data-qa="office">
+      <option value="Tokyo">Tokyo</option>
+      <option value="Singapore">Singapore</option>
+      <option value="London">London</option>
+      <option value="New York">New York</option>
+      <option value="Edinburgh">Edinburgh</option>
+      <option value="San Francisco">San Francisco</option>
+    </select>
+  </label>
+  <label>Age:
+    <input name="age" type="number" data-qa="age">
+  </label>
+  <label>Salary:
+    <input name="salary" type="number" data-qa="salary">
+  </label>
+  <button type="submit">Save to table</button>
+  </form>
+`);
+
+const employeeForm = document.querySelector('.new-employee-form');
+const employeeFormFields = [...employeeForm.querySelectorAll('input, select')];
+
+/* ============== CREAT NOTIFICATION CLASS ============== */
+
+const { bottom, right, width } = employeeForm.getBoundingClientRect();
+
+tableContainer.insertAdjacentHTML('afterbegin', `
+  <div class="notifications" style="
+    position: absolute;
+    top: ${bottom + 25}px;
+    left: ${right - width}px;
+    display: grid;
+    justify-items: end;
+    gap: 25px;
+  "></div>
+`);
+
+/* ============== PUSH CLASS NOTIFICATION ============== */
+
+class ValidationError extends Error {
+  constructor(title, description, type, element) {
+    super(...arguments);
+    this.title = title;
+    this.description = description;
+    this.type = type;
+    this.element = element;
+  }
+}
+
+class Notifications {
+  constructor(title, description, type) {
+    this.title = title;
+    this.description = description;
+    this.type = type;
+  }
+
+  show() {
+    const notificationsContainer = document.querySelector('.notifications');
+
+    notificationsContainer.insertAdjacentHTML('afterbegin', `
+      <div class="notification ${this.type}" data-qa="notification">
+          <h2 class="title">${this.title}</h2>
+          <p>${this.description}</p>
+      </div>
+    `);
+
+    const message = document.querySelector('.notification');
+
+    message.style.top = '0px';
+    message.style.left = '0px';
+    message.style.position = 'static';
+
+    setTimeout(() => message.remove(), 1500);
+  }
+}
+
+/* ============== INPUTS ENTRIES VALIDATION ============== */
+
+function checkUserEntries(entries) {
+  const specification = {
+    'name': {
+      rules: (value) => value.length < 4,
+      description: 'Name must contain at least 4 letters',
```

Comment:
I suggest to move all success messages into SuccessMessages object and errors into ErrorsMessages